reference/configuration.rst

```diff
@@ -144,6 +143,35 @@ Citus has two executor types for running distributed SELECT queries. The desired
 
 This parameter can be set at run-time and is effective on the coordinator. For more details about the executors, you can visit the :ref:`distributed_query_executor` section of our documentation.
 
+.. _multi_task_logging:
+
+citus.multi_task_query_log_level (enum)
+$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$
+
+Sets a log-level for any queries which generates more than one task (i.e. which
+hits more than one shard). This is useful during a multi-tenant application
+migration, as one can choose to error or warn for such queries, to find them and
+add a tenant_id filter to them. This parameter can be set at runtime and is
+effective on the coordinator. The default value for this parameter is 'off'.
+
+The supported values for this enum are:
+
+* **off:** Turn off logging any queries which generate multiple tasks (i.e. span multiple shards)
```

Comment:
Yeah I guess it would be out of place to change just one list. Maybe sometime later I can see whether it's worthwhile to update them all.